packages/xpetra/src/CMakeLists.txt

```diff
@@ -37,6 +116,9 @@ IF (${PACKAGE_NAME}_ENABLE_Epetra)
     )
 ENDIF()
 
+MESSAGE("JHU: CMAKE_CURRENT_SOURCE_DIR=${CMAKE_CURRENT_SOURCE_DIR}")
+TRILINOS_CREATE_CLIENT_TEMPLATE_HEADERS(${CMAKE_CURRENT_SOURCE_DIR}/CrsMatrix NOSIERRABJAM)
```

Comment:
Sierra now uses CMake, not bjam, to set up Trilinos' build.  That could be relevant to this `NOSIERRABJAM` option, depending on what it means.